src/battle_ai_main.c

```diff
@@ -2507,7 +2507,6 @@ static s32 AI_CheckBadMove(u32 battlerAtk, u32 battlerDef, u32 move, s32 score)
 
                 if (instructedMove == MOVE_NONE
                   || gMovesInfo[instructedMove].instructBanned
-                  || gMovesInfo[instructedMove].twoTurnMove
```

Comment:
Shouldn't this still check if it's a 2-turn move?

Reply:
I think the idea is for the AI to charge and fire the move in the same turn if instructed so it would make sense to remove it.

Reply:
Two-turn moves are fully banned from Instruct, even if it could be executed it one turn.

Reply:
oh then it even makes more sense to remove it.

Reply:
> oh then it even makes more sense to remove it.

What? It makes more sense to *not* remove it in this case then, replaced by the effect twoTurnMove flag

Reply:
I mean hopefully those moves have `instructBanned` on them already